### DISCUSSION TITLE
Need CirclePanelDemoGeometry.

### DISCUSSION THREAD

**Discussion by ms-somanna** [HandyOrg/HandyControl#930] [Q&A] [answered]
I'm creating a CirclePanel and I want CirclePanelDemoGeometry as it is not available as built-in geometry.
Can you please provide the geometry.

**Comment by ghost1372** [comment, marked as answer]
https://github.com/HandyOrg/HandyControl/blob/2c9ef4e24c9646ea98a72f716d5505cd2c81f2bf/src/Shared/HandyControlDemo_Shared/Resources/Themes/Basic/Geometries.xaml#L7
